traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/postgres.go

```diff
@@ -433,7 +435,12 @@ func postgresLoad(b json.RawMessage) (trafficvault.TrafficVault, error) {
 		log.Infoln("successfully pinged the Traffic Vault database")
 	}
 
-	return &Postgres{cfg: pgCfg, db: db}, nil
+	aesKey, err := readKeyFromFile(pgCfg.AesKeyLocation)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
We should call `aes.NewCipher` here with the key contents just to ensure it's a valid key. Also, I was thinking we might want to store the key data in base64 encoding rather than raw bytes. That would make it a little easier to pass around (for instance in HashiCorp Vault we might want to store it that way as well).